cmake/modules/SwiftConfigureSDK.cmake

```
@@ -172,6 +174,13 @@ macro(configure_sdk_darwin
     "${SWIFT_SDK_${prefix}_ARCHITECTURES}"        # rhs
     SWIFT_SDK_${prefix}_MODULE_ARCHITECTURES)     # result
 
+  # Determine whether this is a simulator SDK.
+  if ( ${xcrun_name} MATCHES "simulator" )
```

The convention in LLVM (and Swift) is not have spaces after the `if` and not to have spaces inside the `(`.
```suggestion
  if(xcrun_name MATCHES simulator)
```

---

I had no idea, thank you!